apps/consumer-client/build.ts

```diff
@@ -1,8 +1,11 @@
 import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
 import { NodeModulesPolyfillPlugin } from "@esbuild-plugins/node-modules-polyfill";
+import * as dotenv from "dotenv";
 import { build, BuildOptions, context } from "esbuild";
 import fs from "fs";
 
+dotenv.config();
```

Comment:
how did this ever work locally? this might not be the right way of getting environment variables in to `consumer-client`, but i don't know what the right one is.

Reply:
What about this is not working as expected? Have you tested it locally?

Reply:
Oh, is it that this app was missing a reference to the `dotenv` package? i think that it worked anyways because of our monorepo setup. Also, didn't we have a script that would prevent this kind of omission in our monorepo at some point?

Reply:
I don't know, I just know that `consumer-client` didn't have any environment variables defined, and I couldn't figure out how they were supposed to be defined. `passport-client` uses a .env file and the `dotenv` package, so I copied that approach here. My comment is just to say that if this was meant to work some other way, I don't know what that other way is.